nano/scripts/nano/nano_state_pda.js

```diff
@@ -0,0 +1,57 @@
+NanoStatePDAClass.inheritsFrom(NanoStateClass);
+var NanoStatePDA = new NanoStatePDAClass();
+var pda_app_template = null;
```

Comment:
I was having trouble with the state objects...maintaining state, ironically. I can see if I can make it a field of the state again. (Although important to point out that this is technically design aesthetic, as the js vm is specific to a single window)